CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.1.1 - 2022-03-07
```

Comment:
I'd suggest we bump to `2.2.0` instead, especially given `max_length` changes.

Reply:
That seems appropriate. I erred on the side of a patch version because there were only a few changes since v2.1.0, but the `max_length` change has enough impact it could bump the minor version to v2.2.0.